floodlight/io/catapultApi.py

```diff
@@ -31,63 +46,63 @@ def saveResponse(response: requests.Response, endpoint: str) -> None:
     else:
         print(f"Error: {response.status_code} - {response.text}")

```

Comment:
Generally we don't want to print error messages to the console but raise an appropriate error. I think in this case you can raise a `ValueError` that states the expected values for the response and the actual values in the response.

Reply:
Done